random-quote-generator/src/App.js

```diff
@@ -1,24 +1,30 @@
-import logo from './logo.svg';
 import './App.css';
+import quill from './quill.png';
+import quill2 from './quill2.png';
 
 function App() {
+
+  function getQuote(e) {
+    alert("You clicked the inkpot!")
+  }
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+      <div className="content">
+        <h1>Thus wrote the Bard...</h1>
+        <div className="centered">
+          <p id="printQuote">All the world’s a stage,<br />
+And all the men and women merely players</p>
+          <p id="author">- William Shakespeare</p>
+          <img id="button" src={quill} alt="new quote button" 
+          onMouseOver={e => (e.currentTarget.src=quill2)}
+          onMouseOut={e => (e.currentTarget.src=quill)}
```

Comment:
Hey Herman 👋 
Thanks for this implementation, it's definitely a great start.

To let this code look more React-ish, you could try this:
```js
const [ imageSrc, setImageSrc ] = useState( quill ); // at the top of the App function
...
return (
    ...
    <img id="button" src={ imageSrc } ...
        onMouseOver={ () => setImageSrc( quill2 ) }
        onMouseOut=( () => setImageSrc( quill ) }
        ...
```

Reply:
Thanks. I didn't look at your code, wanting to figure it out on my own first, but ended up with basically the same thing :)

I also added a static array with a few quotes for now, and changing the state to let the quotes update when clicking the button.